directory_forms_api_client/version.py

```diff
@@ -1 +1,5 @@
-__version__ = '4.1.0'
+import pkg_resources
+
+__version__ = pkg_resources.get_distribution(
+    'directory-forms-api-client'
+    ).version
```

Comment:
this file can be deleted. to clarify the idea behind [this](https://github.com/uktrade/directory-cms-client/pull/44/files#diff-2eeaed663bd0d25b7e608891384b7298):
 - delete version.py
 - in all places that currently use `__version.py__`, instead, do `pkg_resources.get_distribution(__package__).version`
e.g, 
![image](https://user-images.githubusercontent.com/5485798/61280629-1c285a00-a7b0-11e9-8d2b-33fe0ea23c40.png)